index.html

```diff
@@ -496,21 +493,20 @@ <h2>
         unexpected behavior, use a scope ending in a <code>/</code>.
       </div>
       <p>
-        If the <a>application context</a>'s <a>active document</a>'s
-        [=Document/URL=] is not <a data-lt="within-scope-manifest">within
-        scope</a> of the <a>application context</a>'s manifest, the user agent
-        SHOULD show a prominent UI element indicating the [=Document/URL=] or
-        at least its <a>origin</a>, including whether it is served over a
-        secure connection. This UI SHOULD differ from any UI used when the
-        [=Document/URL=] is <a>within scope</a>, in order to make it obvious
-        that the user is navigating off scope.
+        If the [=application context=]'s [=active document=]'s [=Document/URL=]
+        is not [=manifest/within scope=] of the [=application context=]'s
+        [=processed manifest=], the user agent SHOULD show a prominent UI
+        element indicating the [=Document/URL=] or at least its [=origin=],
+        including whether it is served over a secure connection. This UI SHOULD
+        differ from any UI used when the [=Document/URL=] is [=manifest/within
+        scope=] of the [=application context=]'s [=processed manifest=], in
```

Comment:
Echoing a previous comment: feel free to disagree here, it's a very minor nit: I would prefer to not say "processed manifest" everywhere, since it's implied by "within scope" that the manifest be processed.

I think it would read more cleanly to just say "application context's manifest". Similarly down below for deep links, and shortcut's `url` item, you could say "manifest" instead of "processed manifest".

Reply:
I think I'm going to stick with "processed manifest" for these ones if that's ok. Even though you are right that it's implied by "within scope", it reminds the reader that it's not the unprocessed one that was downloaded.